Protocol.md

```diff
@@ -1101,6 +1102,33 @@ key pair and the other client's session key pair.
 }
 ```
 
+## 'application' Message
+
+Once the client-to-client handshake has been completed, the user
+application of a client MAY trigger sending this message.  
+This message allows user applications to send simple control
+messages/early data or whatever it desires without having to modify an
```

Comment:
How about this?

  This message type allows user applications to send simple control messages or early data without having to modify an existing task.